javalib/src/main/scala/java/nio/GenBuffer.scala

```diff
@@ -17,6 +17,11 @@ import java.util.internal.GenericArrayOps._
 private[nio] object GenBuffer {
   def apply[B <: Buffer](self: B): GenBuffer[B] =
     new GenBuffer(self)
+
+  @inline def validateAllocateCapacity(capacity: Int): Unit = {
+    if (capacity < 0)
+      throw new IllegalArgumentException
```

Comment:
This one doesn't have a message like `explicitlyValidateCapacity`.

Reply:
I did it that way in the javalib to replicate what I observed on the JVM.

Reply:
That was my only possible guess - you don't miss much - at least someone is interested.